solutions/cses-1132.mdx

```diff
@@ -80,7 +83,98 @@ int main() {
 	}
 	return 0;
 }
+```				 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java exceeds the time limit on two test cases.
```

Comment:
... . This is unavoidable because Java has an inherently slow speed compared to C++ and other languages

Reply:
people already assume this. this is the same format as other java warnings